CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ## Unreleased
 
 ### Breaking changes
--
+
+### Compatible changes
+- Add `rubocop-rspec-rails`
```

Comment:
Hmm.. it's hard to decide what makes a change in Rubocop "breaking". Actually any new cop. Let's increment MAJOR.